pkg/reconciler/openshift/tektonaddon/pipelinesascode.go

```diff
@@ -86,3 +94,12 @@ func (r *Reconciler) ensurePAC(ctx context.Context, ta *v1alpha1.TektonAddon) er
 
 	return nil
 }
+
+// getPACImage gets the PAC image name from the IMAGE_PAC_PIPELINES_AS_CODE env var
+func getPACImage() (string, error) {
+	pacImage := os.Getenv(common.PACImageEnv)
+	if pacImage == "" {
+		return "", fmt.Errorf("environment variable %v not found", common.PACImageEnv)
```

Comment:
is it required to return an error here?
if image not found then we can continue with existing img right?
what happens in development case? when we do `make apply` locally
do we have a image in env?

Reply:
good catch, fixed in https://github.com/tektoncd/operator/pull/664/commits/1d5be1021ef4faedc26493b48628efb90e6012b8